backend/collaboration-service/app/src/constants/language.ts

```diff
@@ -0,0 +1,14 @@
+export const LANGUAGES = {
```

Comment:
From question-service, possible languages are:
```
"languages": [
      "bash",
      "c",
      "cpp",
      "csharp",
      "dart",
      "elixir",
      "erlang",
      "golang",
      "html",
      "java",
      "javascript",
      "kotlin",
      "mssql",
      "mysql",
      "oraclesql",
      "php",
      "python",
      "python3",
      "pythondata",
      "pythonml",
      "racket",
      "react",
      "ruby",
      "rust",
      "scala",
      "swift",
      "typescript",
      "vanillajs"
    ]
```

Code snippets in questions are in the format:
```
{
      "_id": "652a0c140ba8294af837eec1",
      "id": 1,
      "category": "Algorithms",
      "code_snippets": [
        {
          "lang": "C++",
          "langSlug": "cpp",
          "code": "class Solution {\npublic:\n    vector<int> twoSum(vector<int>& nums, int target) {\n        \n    }\n};"
        }, ...
```

where `lang` refers to the formal Language title and `langSlug` refers to the slug for the language as denoted in `languages`

Reply:
merge for now as the code sample integration is not in the current feature scope